site/src/atomics/shadow.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Shadow Atomics
+description: Shadow related atomic css classes for the Atlas Design System
+template: standard
+---
+
+# Shadow Atomics
+
+Shadow atomics are used to add a shadow effect around the frame of an element.
+
+## Usage
+
+Here are examples of shadow atomics, ranging from `none` for no shadow to `extra-heavy` shadow.
+
+```html
+<div class="display-block border padding-block-sm">
```

Comment:
Replace with `padding-sm` so the shadow on the left and right sides is also visible